contents/docs/self-host/runbook/clickhouse/resize-disk.md

```diff
@@ -26,7 +29,19 @@ showTitle: true
     /dev/disk/by-id/scsi-0DO_Volume_pvc-f39035c1-c68c-4572-81f2-273de6eb088c   40G  186M   38G   1% /var/lib/clickhouse
     ```
 
-Note: this procedure doesn't work to decrease a volume. If you try, the disk won't be resized and the following errors can be seen in the ClickHouse operator pod
+#### Troubleshooting
+
+If the resize didn't work check for errors in the ClickHouse operator pod.
```

Comment:
Why? What the ClickHouse operator pod will show us?

Reply:
iirc: All the errors go there for CH resize attempts - e.g. when you try a helm upgrade with a downsize it will report everything was fine, but the logs show up in CH operator pod.